packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -67,7 +68,9 @@ export type TooltipProps = (ManagedModeProps | UnmanagedModeProps) &
     TooltipBoxProps;
 
 export const Tooltip = ({
+    isActive = true, // NOTE: determines, if the tooltip is actually working - is displayed, reacts to the events
```

Comment:
I suggest remove comment here, IMHO it's enough to have just one in `TooltipUiProps`